New paragraph added

ssfreitas opened the pull request:
New paragraph

robertovormittag commented on index.html:
```diff
@@ -12,6 +12,7 @@
 <h1>Open Website</h1>
 
 <p>Let's make this a great website!</p>
+<p>I am havinhg fun with Git and GitHub!</p>
```
Please correct mispelling: havinhg -> having
Thank you!

ssfreitas commented:
Good Morning, Roberto.
I have corrected my misspelling, added and committed the file, and pushed the topic branch to my fork. 
However, when I click in "New Pull Request", I see my 2 commits, but no option to create a new pull request.
I´d like to know if it´s all right. In other words: as I made a first pull request yesterday, I don´t need to make another one and you'll be able to merge the file.
Thank you.
By the way, your book is 5 stars. I rated it yesterday at Amazon Brazil.

robertovormittag commented:
Hi Sergio,
That's correct, the pull request is already open and you do not need to create a new one. I was able to merge and you can now see your change in the main code base in master. Many thanks for your kind review, I am glad you found the book to be useful.
